lms/db/__init__.py

```diff
@@ -28,34 +28,6 @@ def columns(cls):
             if isinstance(property, ColumnProperty)
         ]
 
-    def update_from_dict(self, data, skip_keys=None):
```

Comment:
IMO this method promotes and antipattern, it's difficult to find exactly what is changing here.

Reply:
I agree, this is just unnecessary cleverness and makes code harder to follow than necessary, and an unwillingness to just people simple and direct. Restraint is good for developers to learn.